front/app/containers/Authentication/steps/Policies/FranceConnectLogin.tsx

```diff
@@ -14,12 +16,14 @@ const FranceConnectLogin = ({ onLogin }: Props) => {
   const { formatMessage } = useIntl();
 
   return (
-    <FranceConnectButton
-      logoAlt={formatMessage(oldMessages.signUpButtonAltText, {
-        loginMechanismName: 'FranceConnect',
-      })}
-      onClick={onLogin}
-    />
+    <Suspense fallback={null}>
```

Comment:
Also, why did you focus on these files in particular? These buttons are fine to dynamically import but I doubt that they are going to significantly change the lighthouse score